src/main/java/com/google/sps/servlets/CronServlet.java

```diff
@@ -114,16 +113,17 @@ private void resetPostedYet() {
             String description = (String) deed.getProperty(DESCRIPTION);
             String posted_yet_string = (String) deed.getProperty(POSTED_YET);
             boolean posted_yet_bool = Boolean.parseBoolean(posted_yet_string);
+            String link = (String) deed.getProperty(LINK);
             long timestamp = (long) deed.getProperty(TIME_STAMP);
  
-            GoodDeed deedObject = new GoodDeed(key, id, title, description, posted_yet_bool, timestamp);
+            GoodDeed deedObject = new GoodDeed(key, id, title, description, posted_yet_bool, timestamp, link);
             GoodDeeds.add(deedObject);
         }
 
         return GoodDeeds;
     }
 
-    private void selectDailyDeed(List<GoodDeed> GoodDeeds) {
+    public void selectDailyDeed(List<GoodDeed> GoodDeeds) {
```

Comment:
fyi - it is usually not a good idea to change make a `private` method `public`, just so that we can test it. A slightly better way is to remove the visibility modifier altogether so that it is available in tests. But typically we shouldn't be testing `private` methods as they are internal to the class. We can verify the result of the public method uses them.